### ISSUE TITLE
[Local Evento] Orçamento Local Evento.

### ISSUE THREAD

**Issue by jailsonpj** [PythonNorteBrasil/pynorte2025-org#3]
Solicitar orçamento das salas do Hotel Sagres em Belém para dois dias de eveno, sábado e domingo. Nas datas 04, 05 e 06/07 de 2025.

site: https://www.hotelsagres.com/eventos

**Comment by thalisonwilker** [comment]
Passei na frente do hotel  Sagres e resolvi parar pra olhar o orçamento. Fui informado pela recepção que a agenda de 2025 só vai poder se consultada a partir de dezembro de 2024, mas me informaram os valores de diárias de 2024:

Salão Atlântico com capacidade de até 400 pessoas: diária R$ 6.600,00

Salão Amazonas, com capacidade de até 200 pessoas, diária R$ 3.850,00

A recepção também me informou que a diferença de orçamento para 2025 seria de no máximo 10% de aumento.

![image](https://github.com/user-attachments/assets/0cee2da3-968e-4f33-9750-7b18534a232d)

**Comment by jailsonpj** [comment]
Outro Hotel para pedidode orçamento: https://www.princesabelem.com.br/#EVENTOS

**Comment by jailsonpj** [comment]
Estação das Docas, outro local para pedido de orçamento: Estação das Docas
https://estacaodasdocas.com/#seueventoconosco

**Comment by thalisonwilker** [comment]
"A data de julho temos disponibilidade..."

Você deve solicitar o orçamento via e-mail no comercial@para2000.com.br
Informar o nome do evento e a data pretendida, assim como os espaços que deseja alugar

Envio para você uma apresentação do Hangar e Estação das Docas para melhor conhecimento.

[Estação das Docas - Cultural e Empresarial](https://github.com/user-attachments/files/17619407/Estacao.das.Docas.-.Cultural.e.Empresarial.pdf)

[Hangar - Congressos e Feiras](https://github.com/user-attachments/files/17619415/Hangar.-.Congressos.e.Feiras.pdf)

**Comment by thalisonwilker** [comment]
Princesa Louça (preços 2024)
Auditório com até 200 pessoas, diária R$ 4.300,00 (Uso por 10h)
Auditório com até 500 pessoas, diária R$ 8.800,00 (Uso por 10h)

**Comment by thauska** [comment]
Pessoal, podemos também considerar espaço cedido em instituição de ensino público e privado? Até pra ver questão de laboratório de informática.

**Comment by thaty-maru** [comment]
@thauska sim, inclusive essa possibilidade foi levantada na reunião de ontem. Para instituições públicas seria para fazer tutoriais e sprints gratuitas, e no espaço privado para a parte paga (palestras, keynotes, etc)

**Comment by thauska** [comment]
Ótimo. Estou junto ao líder do centro acadêmico da UEPA verificando se tem a possibilidade de ser em algum centro da UEPA. Mantenho vcs informados por aqui.

**Comment by thalisonwilker** [comment]
@thauska eu fiz uma pequena pesquisa de algumas opções para os locais de tutoriais e o evento :) e também me informaram a possibilidade de parceria com as instituições particulares, mas também me informaram que algumas delas (UNAMA e CESUPA) o auditório não funciona aos domingos.

Para o evento Pago
- Sagres
- Princesa louça
- Hangar
- Ccbeu

Para os tutoriais
- Unama
- CESUPA
- UFPA
- UFRA

**Comment by thauska** [comment]
Para os tutoriais, existe a possibilidade de solicitar tbm espaço nas
Usinas da Paz. Entrei em contato com um rapaz que trabalha na SEAC (
https://www.seac.pa.gov.br/) e ele disse para criar um ofício solicitando o
espaço. Ele recomenda as Usinas do Bengui ou Jurunas que têm estruturas
boas nos laboratórios de informática.

**Comment by theunrealryan** [comment]
Olá galera, sou o Ryan do Centro Acadêmico de Engenharia de Software (CAESoft) da Universidade do Estado do Pará (UEPA) Campus XXII Ananindeua. Já realizamos eventos da Owasp e Devs Norte/AWS User Group no Auditório do CCNT [https://g.co/kgs/1R4cYDs](https://g.co/kgs/1R4cYDs) que foi reformado em 2022 e comporta cerca de 250 pessoas, o qual não é cobrado valor devido ser solicitado através da Coordenação do Curso como evento de Eng. Software em parceria com a comunidade de tecnologia. 

Verifiquei com o coordenador do curso o qual recomendou ser em provável período letivo devido precisar de alguém da coordenação do curso para também acompanhar e estar presente no evento assim como nos anteriores, a Assessoria de Planejamento do CCNT relatou que devido a antecedência é tranquilo pra agendar para os dias 07, 08 e 09 (quinta, sexta e sábado) de agosto de 2025 os quais também particularmente considero melhores datas para termos melhor adesão ao evento pois a maioria das pessoas viaja em julho. Apenas não conheço ao certo os laboratório de informática do CCNT devido não ser o meu campus de aula, mas o campus onde fica a coordenação do curso a qual a nova direção está super solícita com as propostas de eventos e parcerias através do centro acadêmico. 

Dessa forma através do centro acadêmico com a coordenação do curso dá pra conseguir o auditório do CCNT da UEPA de quarta-sexta ou quinta-sábado na primeira semana de agosto de 2025.

**Comment by thalisonwilker** [comment]
Conversei ontem com o centro acadêmico de sistema de informação da UFPA, me repassaram algumas informações e ficaram animados com a oportunidade de parceria.

"Já adiantando basicamente como é o espaço na ufpa, é assim: os laboratórios ficam no nosso instituto o ICEN e atualmente a gente tem 3 laboratórios que são equipados com uns 30 computadores, como o evento provavelmente vai ser no período de recesso, acredito que vai ser tranquilo pra disponibilizar esses laboratórios.
Caso precise de mais, podemos ver no itec os laboratórios que possam alugar, o único problema é que são prédios diferentes, mas aí já soma uns 5 laboratórios.
Acho que é super possível fazer o evento nos dois institutos, o ICEN e o ITEC, que no caso é ciência/sistemas e o ITEC é de engenharia da computação"

Também conversei com o centro acadêmico de computação da UFRA e também gostaram da proposta.
"Temos o pavilhão de salas de aula que o acesso é mais fácil, são aproximadamente 45 PCs e são 4 laboratórios."

Fiquei de retornar o contato no máximo até a próxima quarta para oferecer a proposta de cota para apoiadores, caso o material de patrocínio já esteja pronto até lá. Se não eu uso os da edição anteriores como base.

De modo geral, temos também as opções de laboratórios:

- ICEN, ITEC (UFPA)
   - 3 labs, 30 máquinas
- CALC (UFRA)
   - 4 labs, +- 45 máquinas

**Comment by thalisonwilker** [comment]
Hotel Princesa Louçã ( +55 91 9309-0051, eventos@princesabelem.com.br ):
Orçamento para as datas 5 e 6 de julho (Sábado e domingo) o Salão Carajás, até 
Diária R$ 10.200,00 10h de utilização

Esta área é o foyer do Karajás
![image](https://github.com/user-attachments/assets/212700a8-79ce-4c65-ace1-fe2199df3cd1)
![image](https://github.com/user-attachments/assets/e7ccf072-e9fb-4bcf-b0d7-ff98772b5b71)
![image](https://github.com/user-attachments/assets/e2a28666-bb1b-42f4-94a4-56f851ab1410)


Auditório, o salão fica localizado no 2º subsolo:
![image](https://github.com/user-attachments/assets/a416c67a-44e1-4f92-84ec-dea3031a6ad9)


[Coffee Break - Hotel Princesa Louçã.pdf](https://github.com/user-attachments/files/18333522/Coffee.Break.-.Hotel.Princesa.Louca.pdf)

**Os valores de alimentação serão mantidos para eventos confirmados em contrato até o dia 31/01. Após esta data teremos reajuste nos valores**

**Comment by thalisonwilker** [comment]
o auditório Karajás tem capacidade de até 300 lugares, área para Coffee Break e expositores

**Comment by thauska** [comment]
Bom dia, povo

Pedi orçamento na Estação das Docas. Enviaram esse PDF com as informações:

![Estação das Docas - Eventos Sociais.pdf](https://github.com/user-attachments/files/18571829/Estacao.das.Docas.-.Eventos.Sociais.pdf)
